covid19_sir/model/base.py

```diff
@@ -6,6 +6,11 @@
 from mesa.time import RandomActivation
 from model.utils import TribeSelector, SimulationState, DilemmaDecisionHistory, WeekDay
 
+def print_world(model):
+    for agent in model.agents:
+        if isinstance(agent, District): 
```

Comment:
We shouldn't add dependencies of humans, locations etc in `base.py` otherwise we'll end up with painful cyclic dependency.

Reply:
I created a `debugutils.py` which have several lists of entities like:

```
         self.humans = []
        self.adults = []
        self.k12students = []
        self.toddlers = []
        self.infants = []
        self.elders = []
        self.locations = []
        self.districts = []
        self.restaurants = []
```

They are populated using `CovidModel`. I think we may add debug methods to this file instead of putting than in the model so we keep our codebase cleaner.

I've created `print_world()` there already (although I've used your original code without the fix I mentioned in https://github.com/singnet/covid-simulation/issues/61